website/blog/four-approaches-for-documenting-apis.md

```diff
@@ -0,0 +1,126 @@
+---
+date: "06/09/2021"
+title: "Four Approaches for Documenting APIs"
+author: Deborah Ruck
+author_url: "https://deborahruck.com/"
+author_image_url: "/img/people/deborah-ruck.jpeg"
+category: Community
+---
+
+According to Gallup, my top strength is [Learner](https://www.gallup.com/cliftonstrengths/en/252293/learner-theme.aspx). That’s probably the reason I got into software development over two decades ago. Technology changes so quickly, there’s always something new to learn. 
+
+Java was the first real programming language I used back then. At the time, it had been around for about three years and was still very immature. But all this time, there’s something that I have always remembered and appreciated about Java: its exceptional API documentation. It was easy to understand, the examples were useful, and it was usually all I needed to solve any problem. 
+
+Good API documentation shouldn’t be exceptional, but it is. And it’s important.
+
+<!--truncate -->
+
+Your API documentation shows consumers how best to use your API. Documentation that’s not helpful or easy to understand (or worse yet, no documentation at all) will hinder the adoption of your API, no matter how good it is. 
+
+But creating API documentation and keeping it up to date can be a challenge. In many cases,  the only documentation available consists of ad hoc comments left by the developer who last touched the code. 
+
+In this post, we’ll compare four approaches you can use to create and maintain your API documentation:  
+* Manually updating API docs
+* Designing API docs then building to it, (3) programmatically generating API docs with Express OpenAPI, and (4) using Optic to automatically diff and update your API docs.
```

Comment:
Strong 1, b, third energy in this list. Replaced with numbered list to match the method numbering below in upcoming commit